ServerCore/ModelBases/EventSpecificPageModel.cs

```diff
@@ -32,24 +32,19 @@ public PuzzleUser LoggedInUser
             {
                 if (loggedInUser == null)
                 {
-                    loggedInUser = PuzzleUser.GetPuzzleUserForCurrentUser(puzzleServerContext, User, userManager).Result;
+                    loggedInUser = PuzzleUser.GetPuzzleUserForCurrentUser(_context, User, userManager).Result;
                 }
 
                 return loggedInUser;
             }
         }
 
-        private readonly PuzzleServerContext puzzleServerContext;
+        protected readonly PuzzleServerContext _context;
```

Comment:
Should this be a property since it's protected (similar to the one you've removed from PuzzleStatePerTeamPageModel?)

Reply:
Yes but I was trying to keep change volume and merge conflicts low. We should address this once other sweeping changes are in.